go.mod

```diff
@@ -54,4 +54,6 @@ require (
 // Workaround for github.com/containerd/containerd issue #3031
 replace github.com/docker/distribution v2.7.1+incompatible => github.com/docker/distribution v2.7.1-0.20190205005809-0d3efadf0154+incompatible
 
+replace github.com/firecracker-microvm/firecracker-go-sdk => github.com/RoyceDavison/firecracker-go-sdk v0.22.1-0.20210402232445-4bb6cbb0e7f8
```

Comment:
hm should this be here?

Reply:
Yeah, this shouldn't be here. As my commit message suggested "add go.mod go.sum for integ tests, will remove it later"